tests/start_from_markduplicates.nf.test.snap

```diff
@@ -610,9 +597,7 @@
                 "test.recal.mosdepth.region.dist.txt:md5,75e1ce7e55af51f4985fa91654a5ea2d",
                 "test.recal.mosdepth.summary.txt:md5,b23cf96942b2ada3f41172a9349a1175",
                 "test.recal.regions.bed.gz:md5,74cd0c779c7b3228adcf3b177333886a",
-                "test.recal.regions.bed.gz.csi:md5,080731cdedcd389e72135f048d6e2e00",
-                "test.md.cram.stats:md5,f181d98f08ad94c3926ac149a87d834b",
-                "test.recal.cram.stats:md5,18346c938c7b1bfaf9ac9413fdba90d8"
```

Comment:
what happened to the checksums?

Reply:
Famke removed some cram.stats, because they were unstable, so I removed them all